ldms/src/contrib/store/timescale/Makefile.am

```diff
@@ -0,0 +1,24 @@
+SUBDIRS =
+lib_LTLIBRARIES =
+pkglib_LTLIBRARIES =
+dist_man7_MANS =
+dist_man1_MANS =
+
+AM_LDFLAGS = @OVIS_LIB_ABS@
+AM_LDFLAGS += -L/usr/lib64 -lpq -lecpg
```

Comment:
It is not correct to hard code the "/usr/lib64" path in AM_LDFLAGS. And for a library, these things should be in LIBADD, not LDFLAG.

One way this can be resolved is by using hte AC_LIB_HAVE_LINKFLAGS macro (which is now included in our repo's m4 macros directory). You will need one call to check for libpq, and another for libecpg.  If you the macro suggested, then you will get two variables that you can use in the LIBADD line: LTLIBPQ and LTLIBECPG. These two macros take care of both the -L and -l options.  So you can would then just add "$(LTLIBPQ) ${LTLIBECPG)" to the libstore_timescale_la_LIBADD line.

Reply:
Hi @morrone, thank you for the suggestion, but I'm not sure how to use the macro and make the call as you mentioned. Could you please give some more instructions?

Reply:
Take a look in configure.ac for an example for how AC_LIB_HAVE_LINKFLAGS is used for papi and pfm. You would do something very similar in configure.ac for the pq and ecpg libraries.

For full documentation of the AC_LIB_HAVE_LINKFLAGS macro and what its options are, take a look here: https://www.gnu.org/software/gnulib/manual/html_node/Searching-for-Libraries.html

When that macro is used for both of the libraries, the libadd line in this file should look like:

libstore_timescale_la_LIBADD = $(STORE_LIBADD) $(LTLIBPQ) $(LTLIBECPG)